tests/setData.spec.ts

```diff
@@ -123,4 +123,43 @@ describe('setData', () => {
       'Cannot add property count'
     )
   })
+
+  it('https://github.com/vuejs/vue-test-utils-next/issues/538', async () => {
+    const Comp = defineComponent<
+      {},
+      {},
+      { field: number | null },
+      { isFieldNull: any }
```

Comment:
Unfortunately the type for this is not exported. I thought we had some way to type `computed` properties and `vm` 🤔

Reply:
I don't use the Options API, so I don't know... Shouldn't it be `boolean` though?

Reply:
According to VS Code it should be `Computed<boolean>` or something like that, but it is not exported from Vue. I shouldn't actually need to specify this here, but if I don't TS complains. 

I have not used TS with Options API before so I have no idea 🤷‍♂️